### PR TITLE
Rename config option namespace_suffix to namespace_prefix

### PR DESCRIPTION
**Pull request by dalees**
This option is used as a prefix, not a suffix.
Breaking change, but hopefully before others start to configure it.

### CODE REVIEW

**Comment by dalees** [comment]
Hi @JohnGarbutt - please review this PR sometime.

**Review by JohnGarbutt — APPROVED** [review]
Sorry, I didn't see this till just now.

Thank you, this looks good. Although I was expecting some tests to break :/

**Comment by JohnGarbutt** [comment]
Ah, fun, we need this to merge: https://review.opendev.org/c/openstack/releases/+/897404

**Comment by JohnGarbutt** [comment]
OK, planning to rebase on top of my temp CI fix.

**Comment by JohnGarbutt** [comment]
Ah, this works fine when I merge it locally, lets merge this known the CI will pass after merge.

**Pull request merged**
